src/Tools/CodeGenerator/Featurizers.SimpleSchema

```diff
@@ -24,10 +24,21 @@
 <featurizers +>:
     [name string]
     [estimator_name string]
+
+    [release_version string]
+    [updated_version string ?]
+    [deprecated_version string ?]

```

Comment:
Whats the difference between release_version and updated_version?

Reply:
Release_version is when the featurizer was released, updated_version is the version when the featurizer was last updated after its release.